hera_mc/cm_revisions.py

```diff
@@ -79,6 +87,13 @@ def get_last_revision(hpn, session=None):
         (hpn, rev, rev_query, started, ended, [hukey], [pkey])
 
     """
+    if session is None:
```

Comment:
Does this function (and the other functions in this file) really need this? It doesn't seem like something a user is likely to run as a stand-alone db call to me, but I'm not a CM power user.

Reply:
The revision stuff is my one big regret in the whole CM business and came about from some very early discussions.  These functions are used rarely to never (I think one or two scripts uses one or two).  I'd like to raise a technical debt issue to purge as much of the revision stuff as possible.  In the meantime, it might be best to leave the structure as it is to avoid the unintended.